src/status_im/contexts/wallet/create_account/new_keypair/backup_recovery_phrase/view.cljs

```diff
@@ -56,9 +63,11 @@
          :title           (i18n/label :t/backup-recovery-phrase)
          :description     (i18n/label :t/backup-recovery-phrase-description)}]
        [rn/view {:style (style/seed-phrase-container theme)}
-        [words-column temp/secret-phrase true]
-        [rn/view {:style (style/separator theme)}]
-        [words-column temp/secret-phrase false]
+        (when (pos? (count @secret-phrase))
+          [:<>
+           [words-column @secret-phrase true]
```

Comment:
for situations like this I preferred maps, it's easier to understand what the code is doing this way.

i.e
`[words-column @secret-phrase true]`
what is being set as `true` here, and why? like this I have to check.
with a named arg it becomes clearer to read

```
     [words-column {:secret-phrase @secret-phrase
                                  :first-half? true}]
     [rn/view {:style (style/separator theme)}]
     [words-column {:secret-phrase @secret-phrase
                                  :first-half? false}]])
```

Reply:
Updated 👍